app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,7 @@
+class CommentsController < ApplicationController
+    
+    def create
+        @step = Step.find(params[:step_id])
+        @comment = @step.comments.create(comment_text: params[:comment_text],user_id: params[:user_id])
```

Comment:
Do you understand the difference between create and new?

Reply:
Yeah so new creates an instance (maybe empty or with any attributes) but it does not save the instance to the database. But create does both new and save , it's a combination of create and save methods .